data/salmanakhtar57.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
You have duplicated the first section. Please Decide which one you want to keep and remove the extra.

Reply:
I want to keep the second one and remove the first one.

Reply:
You can remove the other extra one [here](https://github.com/salmanakhtar57/LinkFree/edit/salmanakhtar57-patch-1/data/salmanakhtar57.json) then. Feel free to let us know if you need any help with this! :)

Reply:
Thanks for letting me know. I removed the extra one and commit changes. Let me know if anything else that I need to do.